core/src/test/java/com/eternalcode/core/user/PrepareUserControllerTest.java

```diff
@@ -1,65 +1,88 @@
 package com.eternalcode.core.user;
 
-import be.seeseemelk.mockbukkit.MockBukkit;
-import be.seeseemelk.mockbukkit.MockPlugin;
-import be.seeseemelk.mockbukkit.ServerMock;
-import be.seeseemelk.mockbukkit.entity.PlayerMock;
 import com.eternalcode.core.user.client.ClientBukkitSettings;
 import com.eternalcode.core.user.client.ClientSettings;
-import net.kyori.adventure.text.Component;
-import org.junit.jupiter.api.AfterAll;
-import org.junit.jupiter.api.BeforeAll;
+import org.bukkit.Server;
+import org.bukkit.entity.Player;
+import org.bukkit.event.player.PlayerJoinEvent;
+import org.bukkit.event.player.PlayerKickEvent;
+import org.bukkit.event.player.PlayerQuitEvent;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
 import panda.std.Option;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertInstanceOf;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import java.util.UUID;
 
-class PrepareUserControllerTest {
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
-    private static ServerMock server;
-    private static MockPlugin plugin;
+public class PrepareUserControllerTest {
 
-    @BeforeAll
-    public static void load() {
-        server = MockBukkit.mock();
-        plugin = MockBukkit.createMockPlugin();
-    }
+    private static final UUID FAKE_UUID = UUID.randomUUID();
+    private static final String FAKE_NAME = "Martin";
+
+    private PrepareUserController prepareUserController;
+
+    @Mock
+    private UserManager userManager;
+
+    @Mock
+    private Server server;
 
-    @AfterAll
-    public static void unload() {
-        MockBukkit.unmock();
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
zamykaj mockito, to co zwraca `openMocks(this)` zapisujesz do fielda i zamykasz `.close()`